connector/AUTHORS

```diff
@@ -0,0 +1,25 @@
+* Guewen Baconnier at Camptocamp
```

Comment:
Is there a reason to keep this? If not, could you get rid of this and use `readme` fragments? The only author here is @guewen the rest goes to contributors.

Reply:
I think you're right. I changed that.